fonts.css

```diff
@@ -0,0 +1,23 @@
+@font-face {
+  font-family: "Montserrat";
+  src:
+    url("fonts/montserrat-500-normal.woff") format("woff"),
+    url("fonts/montserrat-500-normal.woff2") format("woff2");
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)